rocm-validation-suite/PKGBUILD

```diff
@@ -10,10 +10,13 @@ license=('MIT')
 depends=('pciutils' 'doxygen' 'rocblas' 'rocm-smi-lib64' 'git')
 makedepends=('cmake')
 options=(!staticlibs strip)
-source=("$pkgname-$pkgver.tar.gz::https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/archive/rocm-$pkgver.tar.gz")
-sha256sums=('17662028a4485b97e3ccaad5e94d20aaa2c3e9e3f741c7ebbf0f8b4cdebcc555')
+source=("$pkgname-$pkgver.tar.gz::https://github.com/ROCm-Developer-Tools/ROCmValidationSuite/archive/rocm-$pkgver.tar.gz" "action.patch")
+sha256sums=('17662028a4485b97e3ccaad5e94d20aaa2c3e9e3f741c7ebbf0f8b4cdebcc555' '16059a24ae9c01a7e77e1323a2437b9222998d3ac19d7c3367bcf821da60ecd9')
 
 build() {
+  cd $srcdir
+  patch ROCmValidationSuite-rocm-3.9.0/rcqt.so/src/action.cpp action.patch 
```

Comment:
You can include the patch before the build function? In particular the prepare function. Take a look at how its done here: https://github.com/rocm-arch/rocm-arch/blob/70f75fcf6b80ed27a2a905c15b73cfec18f3ede9/hsa-rocr/PKGBUILD#L20-L29

It'd be nice if we can follow the existing conventions.